lti_consumer/lti_consumer.py

```diff
@@ -572,7 +572,12 @@ def lti_provider_key_secret(self):
         """
         for lti_passport in self.course.lti_passports:
             try:
-                lti_id, key, secret = [i.strip() for i in lti_passport.split(':')]
```

Comment:
Since we are with Python3 I would prefer to use the smart list unpacking [PEP3132](https://www.python.org/dev/peps/pep-3132/) and add a comment in the code with NOTE prefix with the brief description of the proposed changes. 
The code could be something like this
```
# NOTE(sksankarraj)  <brief description>
lti_id, *key, secret = [i.strip() for i in lti_passport.split(':')]
if not key:
   raise ValueError
key = ':'.join(key)
```

It is also expected you add tests that cover these changes, it seems that [TestProperties](https://github.com/edx/xblock-lti-consumer/blob/master/lti_consumer/tests/unit/test_lti_xblock.py#L71) class will suit fair enough for that.

Thank you in advance.

Reply:
Hi @idegtiarov , Thanks for the review. I think I have fixed it. Could please check again and let me know if any corrections needed.

Reply:
FWIW, I don't like leaving people's names in the comments, since all of it should be collaboratively owned and authored.  If we want to know who put something in the code, we can use "git blame".